### DISCUSSION TITLE
cara ubah sc

### DISCUSSION THREAD

**Discussion by Patrick12a** [Nurutomo/wabot-aq#853] [Q&A]
bang cara ubah sc hide tag gmn? admin gc gua kek tai:v

**Comment by unx21** [comment]
> bang cara ubah sc hide tag gmn? admin gc gua kek tai:v

ubah

handler.admin = true

jadi

handler.owner = true

**Reply by Patrick12a** [reply]
filenya dmana tempatnya bang

**Reply by Patrick12a** [reply]
w cari2 di plugin gaada

**Reply by unx21** [reply]
ada, nama file nya pengumuman.js

**Reply by Patrick12a** [reply]
oaala, w coba bang tq

**Reply by Patrick12a** [reply]
thx bang work

**Reply by unx21** [reply]
> thx bang work

oke

**Reply by Patrick12a** [reply]
berteman kita yu

**Reply by unx21** [reply]
> berteman kita yu

??

**Reply by Patrick12a** [reply]
mri berteman qqq

**Reply by unx21** [reply]
> mri berteman qqq

[:v](https://wa.me/62887436568613)
